test/sanity/code-smell/no-smart-quotes.py

```diff
@@ -15,10 +15,11 @@ def main():
         'test/integration/targets/template/files/encoding_1252_utf-8.expected',
         'test/integration/targets/template/files/encoding_1252_windows-1252.expected',
         'test/integration/targets/template/templates/encoding_1252.j2',
+        'docs/docsite/_build/',
     ])
 
     for path in sys.argv[1:] or sys.stdin.read().splitlines():
-        if path in skip:
+        if any(path.startswith(skip_path) for skip_path in skip):
```

Comment:
Use a separate skip and prune list, as is done in other sanity tests:

https://github.com/ansible/ansible/blob/60a24bbdaae3a17bf20613810f347bfb8283f306/test/sanity/code-smell/boilerplate.py#L7-L14

Then check each separately:

https://github.com/ansible/ansible/blob/60a24bbdaae3a17bf20613810f347bfb8283f306/test/sanity/code-smell/boilerplate.py#L38-L42

Reply:
Sometimes I think it would be a lot quicker, less effort and less context switching if you would have modified this branch with the change you wanted. It's done now.